pkg/kubelet/cm/OWNERS

```diff
@@ -1,12 +1,7 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- Random-Liu
-- dchen1107
-- derekwaynecarr
-- vishh
-- yujuhong
+- sig-node-approvers
 - ConnorDoyle
```

Comment:
this line seems to be forgotten or left intentionally? or should be moved to emeritus_approvers ?

Reply:
Left intentionally, @ConnorDoyle is not a top-level node approver.